.github/workflows/01-build-and-release.yml

```diff
@@ -0,0 +1,138 @@
+          sed -i 's/DEVICE=\\FREEDOS\\BIN\\HIMEMX.EXE/DEVICE=\\JEMM\\JEMMEX.EXE MAXEXT=2097152\nDEVICE=\\JEMM\\JLOAD.EXE \\JEMM\\QPIEMU.DLL/g' fdconfig.sys
+          mv setup.bat setup.bak
+          echo 'LH \HDPMI\HDPMI32I.EXE' > setup.bat
+          echo 'LH \SBEMU\SBEMU.EXE' >> setup.bat
+          echo 'LH \CTMOUSE\CTMOUSE.EXE' >> setup.bat
+          cd /tmp
+          rm JemmB_v584.zip
+          rm HDPMI32i.zip
+          rm ctmouse.zip
+          sudo umount /tmp/mnt
+          mv FD13LITE.img SBEMU-FD13-USB.img
+          xz -k -9e SBEMU-FD13-USB.img
+          mv SBEMU-FD13-USB.img.xz $GITHUB_WORKSPACE/
+      - name: Generate release tag
+        id: tag
+        run: |
+          echo "release_tag=UserBuild_$(date +"%Y.%m.%d_%H-%M")" >> $GITHUB_OUTPUT
+      - name: Release FreeDOS SBEMU USB image
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.tag.outputs.release_tag }}
+          files: |
+            ${{ github.workspace }}/SBEMU-FD13-USB.img.xz
+          body: |
+            <details>
+            <summary>Preparing a bootable USB drive</summary>
+
+            ## Preparing a bootable USB drive
+
+            The USB image can be written to a USB drive or SD card using a tool like [balenaEtcher](https://etcher.balena.io/).
+
+            The advantage of using Etcher is that you don't have to decompress the `.xz` archive first.
+            It will decompress such files automatically, before writing the image to the target drive.
+            </details>
+            <details>
+            <summary>Booting the USB image in a virtual machine</summary>
+
+            ## Booting the USB image in a virtual machine
+
+            You can run the image in a VM with QEMU as follows:
+
+            ```shell
+            unxz SBEMU-FD13-USB.img.xz
+            qemu-system-i386 -drive file=SBEMU-FD13-USB.img,format=raw -device AC97
+            ```
+
+            If you wish to test Intel HDA compatibility instead of ICHx AC'97 compatibility, replace `AC97` with `intel-hda` in the last command above.
+            On Linux, you can include the parameter `--enable-kvm` to run the VM with hardware-assisted virtualization.
+
+            If you prefer to use another hypervisor, such as VirtualBox or VMware, you may have to convert the raw image to a supported VM image format first:
+
+            ```shell
+            unxz SBEMU-FD13-USB.img.xz
+            qemu-img convert -f raw -O vmdk SBEMU-FD13-USB.img SBEMU-FD13-USB.vmdk
+            ```
+
+            **NOTE**: Although VMs can sometimes be useful during development, testing and debugging, you should not rely on those for actual hardware compatibility testing, since the sound cards that the hypervisors emulate are themselves merely approximations of actual hardware, and will not behave like the real thing in every single corner case.
```

Comment:
Feels like all of this would better be put into `README.md` or some other place, instead of putting all of this into the release text all the time. If nothing else, a `.github/RELEASE-NOTES.md` or something that gets sourced here might be better.

Also, once those get outdated, it will be hard to update, and people might not see them, so I'd rather we have a file in the repository that we can link to (and easily update), and the release notes should just have a link to that file for the most up-to-date information.